### PR TITLE
0.2.9 release notes

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
This PR adds release notes for 0.2.9 and fixes a `None` bug in the release note generation.

### CODE REVIEW

File: docs/release/release_0_2_9.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- in bindings logic, check if generator, not generator function (#853)
+
+Bugfixes
+********
+- fix fullscreen crash for test_viewer (#849)
+- fix RangeSlider.rangeChange emit type bug (#856)
+
+API Changes
+***********
+- rename * to current_* and *s to * for certain layer properties in particular `edge_color` and `face_color`(#686)
```
Comment (kne42) [inline docs/release/release_0_2_9.rst]: what does this mean? I'm confused

also it renders strangely with a link, which you can see [here](https://github.com/napari/napari/blob/b12639589143e9eeaed4a9cc4fb94faa878037af/docs/release/release_0_2_9.rst)

Comment (sofroniewn) [reply docs/release/release_0_2_9.rst]: good catch, that wasn't so clear. I've tried making it more explicit

File: docs/release/release_0_2_9.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Bugfixes
+********
+- fix fullscreen crash for test_viewer (#849)
+- fix RangeSlider.rangeChange emit type bug (#856)
+
+API Changes
+***********
+- `edge_color` and `face_color` now refer to colors of all points and shapes
+  in layer, `current_edge_color` and `current_face_color` arnow refer to the
```
Comment (kne42) [inline docs/release/release_0_2_9.rst]: typo "arnow"

Comment (sofroniewn) [reply docs/release/release_0_2_9.rst]: thanks! fixed

**Review by tlambert03 — APPROVED** [review]
looks good to me

**Review by jni — APPROVED** [review]

**Pull request merged**